IdentityAccess/src/Kreta/IdentityAccess/Infrastructure/Symfony/Framework/config/config.yml

```diff
@@ -21,7 +21,7 @@ framework:
         resource: "%kernel.root_dir%/config/routing.yml"
         strict_requirements: ~
     form: ~
-    csrf_protection: ~
+    csrf_protection: false
```

Comment:
The identityAccess only exposes JSON endpoints, so the form submit is not allowed in this BC